portfolio/src/main/webapp/fetch_script.js

```diff
@@ -7,3 +7,12 @@ async function showVideoGame() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = textFromResponse[number];
 }
+
+
+function showTranslation() {
+    const responseFromServer = await fetch('/arabic');
+    const textFromResponse = await responseFromServer.text();
+
+    const factContainer = document.getElementById('fact-container')
+    factContainer.innerText = textFromResponse
```

Comment:
JavaScript will sometimes automatically insert missing semicolons, but it's clearer if you explicitly include them and don't rely on that working perfectly
```suggestion
    const factContainer = document.getElementById('fact-container');
    factContainer.innerText = textFromResponse;
```